templates/Web/Pages/Vue.Grid/src/views/VueGrid.vue

```diff
@@ -0,0 +1,100 @@
+﻿<template>
+  <div>
+    <main id="mainContent">
+      <div class="text-center header">
+        <h1>Param_ProjectName</h1>
+        <p>This is placeholder text. Your web app description goes here.</p>
+        <a
+          href="https://github.com/Microsoft/WebTemplateStudio"
+          class="btn btn-primary my-2"
+        >Link to our Github</a>
+      </div>
+
+      <div class="container">
+        <div class="row justify-content-center py-5">
+          <h1>Bootstrap Grid Template</h1>
```

Comment:
React and Angular has it as "[Bootstrap ReactGrid Template](https://github.com/microsoft/WebTemplateStudio/blob/dev/templates/Web/Pages/React.Grid/src/components/ReactGrid/index.jsx#L67)" and "[Bootstrap AngularGrid Template](https://github.com/microsoft/WebTemplateStudio/blob/dev/templates/Web/Pages/Angular.Grid/src/app/app-shell/AngularGrid/grid.component.html#L15)" respectively, too.

Reply:
Oops sorry i forgot about this good call